### PR TITLE
Bgs 2.2

### PR DESCRIPTION
**Pull request by bwuphan**

### CODE REVIEW

File: _data/processes.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  description : "We’ll show you a high-level breakdown of your application’s structure and offer
+                 options which might work well for you. You’ll likely have more questions, and we’ll
+                 address those in this meeting.  We’ll do any final sizing and scoping after that.
+                 The option you like best will result in a fully researched final quote and we’ll
+                 sign a contract."
+- title       : "Step 4: Wireframes"
+  image       : 4_wireframes.png
+  description : "A wireframe is a term in our business for the visual representation of the pages in
+                 your application: Where elements go, what they do, what they’ll do next. This shows
+                 you the basic structure of your application. This is the” look and feel” part of
```
Comment (aajungroth) [inline _data/processes.yml]: Adjust spacing:
the "look and feel"
vs
the" look and feel"

File: _sass/_variables.scss [hunk 2]
```diff
@@ -80,12 +80,16 @@ $color-transition-speed: .25s;
...
-
+$px-size-sm: .75rem; /* The padding sides size for small views. */
 $px-size-md: 3%; /* The padding sides size for medium views. */
-
 $px-size-xl: 15%; /* The padding sides size for extra large views. */
 
+/* Padding sizes: Use if you want a smaller container to match the blue headers. */
+$sm-px-size-sm: .75rem; /* The padding sides size for small views. */
+$sm-px-size-md: 6%; /* The padding sides size for medium views. */
+$sm-px-size-lg: 10%; /* The padding sides size for medium views. */
```
Comment (aajungroth) [inline _sass/_variables.scss]: Change medium on line 90 to large.

File: _data/processes.yml (hunk 1 above)
Comment (bwuphan) [reply _data/processes.yml]: Thanks for the catch. Fixed.

File: _sass/_variables.scss (hunk 2 above)
Comment (bwuphan) [reply _sass/_variables.scss]: Fixed.

File: case-studies/bayren.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+title: Bay Area Regional Energy Network (BayREN)
+
+tags: [Our Work]
+
+description: "An example of our work – an eTool - for the Bay Area Regional Energy Network."
+
+client-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
+
+challenge-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
```
Comment (aajungroth) [inline case-studies/bayren.html]: Should this be different from the client-text?

File: case-studies/bayren.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+technologies: [javascript, angular, node, docker, kubernetes, mysql]
+
+outcome-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
```
Comment (aajungroth) [inline case-studies/bayren.html]: Also this. Should this be different from the client-text?

File: case-studies/bayren.html (hunk 3 above)
Comment (bwuphan) [reply case-studies/bayren.html]: Fixed.

File: case-studies/bayren.html (hunk 4 above)
Comment (bwuphan) [reply case-studies/bayren.html]: Fixed.

File: case-studies/bayren.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+
+technologies: [javascript, angular, node, docker, kubernetes, mysql]
+
+outcome-text: "We built a set of tools which takes AutoCAD files, identifies the layers that show certain required information like booth dimensions that, when uploaded, are saved as sets of vertices and redrawn dynamically. We built an online app to show what the floor plan and booth spaces would look like ahead of each event. We were able to create these custom floor plans by manipulating CAD files and sets of vertices which was also rendered on the front-end. This floorplan application allowed buyers to know the “address” of a booth they wanted to visit. The dynamic display would instantly update who moved, who is at this spot or that booth. It made it easy for buyers at market to get an overall look at the layout of the floor. Here’s an example: If someone cancelled from appearing at market their booth would be removed right away so buyers intending to visit it wouldn’t be disappointed."
```
Comment (aajungroth) [inline case-studies/bayren.html]: This line looks like it's from the case study on WEM.

Comment (bwuphan) [reply case-studies/bayren.html]: That's a good point. That's what it said in the spec but probably a mistake on John's part. I'll ask him for the correct text.

Comment (bwuphan) [reply case-studies/bayren.html]: Updated with the correct text.

File: case-studies/bayren.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+
+technologies: [javascript, angular, node, docker, kubernetes, mysql]
+
+outcome-text: We designed and developed an eTool for the areas within BayREN .The homeowner uses the app to fill out details about the project and if their jurisdiction is participating in the program, they get a custom note to fill in their permit application, or get links to the correct permitting information based on their individual project. Customized a pdf download which customized the form.<br><br>Our application streamlines and clarifies the permitting process, and helps educate homeowners and contractors in choosing retrofits that match state and local regulations. Deliverables for this project included wireframes, training materials, and support for both expanded development and implementation of the eTool (including pre- and post-deployment)."
```
Comment (aajungroth) [inline case-studies/bayren.html]: adjust spacing:
within BayREN .The homeowner
should be:
within BayREN. The homeowner

this sentence needs a subject:
Customized a pdf download which customized the form.
I think 'We' should be added to the front:
We customized a pdf download which customized the form.

Comment (bwuphan) [reply case-studies/bayren.html]: Fixed typos.

**Pull request merged**